com.kaii.hgui.json

```diff
@@ -0,0 +1,50 @@
+{
+    "app-id" : "com.kaii.hgui",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "45",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "hgui-rust",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "build-args" : [
+        ],
+        "env" : {
```

Comment:
Set `CARGO_HOME` too

Reply:
fixed all of them in the last commit

Reply:
Not this https://github.com/flathub/flathub/pull/4553/files#r1337704164